### PR TITLE
[Translations] added translations to admin user form

### PR DESCRIPTION
**Pull request by starspire**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

Hi,
it's simple commit adding translations to admin user form (firstName and lastName fields, which was hardcoded earlier).

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Form/Type/User/AdminUserType.php [hunk 1]
```diff
@@ -28,8 +28,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         parent::buildForm($builder, $options);
 
         $builder
-            ->add('firstName')
-            ->add('lastName')
+            ->add('firstName', null, [
+            	'label' => 'sylius.ui.first_name'
+            ])
+            ->add('lastName', null, [
```
Comment (Zales0123) [inline src/Sylius/Bundle/CoreBundle/Form/Type/User/AdminUserType.php]: Shouldn't this and above be `TextType::class`?

File: src/Sylius/Bundle/CoreBundle/Form/Type/User/AdminUserType.php [hunk 2]
```diff
@@ -28,8 +28,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
+            	'label' => 'sylius.ui.last_name'
```
Comment (Zales0123) [inline src/Sylius/Bundle/CoreBundle/Form/Type/User/AdminUserType.php]: Missing comma at the end of line. As well as above.

**Comment by starspire** [comment]
@Zales0123 , thanks. Improved according to your review.

File: src/Sylius/Bundle/CoreBundle/Form/Type/User/AdminUserType.php [hunk 3]
```diff
@@ -28,8 +29,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         parent::buildForm($builder, $options);
 
         $builder
-            ->add('firstName')
-            ->add('lastName')
+            ->add('firstName', TextType::class, [
+            	'label' => 'sylius.ui.first_name',
```
Comment (pjedrzejewski) [inline src/Sylius/Bundle/CoreBundle/Form/Type/User/AdminUserType.php]: We have separate set of translation keys for forms - this should be: ``sylius.form.admin_user.first_name``.

Comment (starspire) [inline src/Sylius/Bundle/CoreBundle/Form/Type/User/AdminUserType.php]: @pjedrzejewski Are you sure? I see `sylius.form.user.first_name` but no `sylius.form.admin_user.first_name`.

Comment (pjedrzejewski) [inline src/Sylius/Bundle/CoreBundle/Form/Type/User/AdminUserType.php]: You are right, should be ``sylius.form.user.first_name``. :)

**Comment by starspire** [comment]
@pjedrzejewski I changed translations according to your comments. Could you rerun travis? The failed test shouldn't have anything in common with my commit.

**Comment by michalmarcinkowski** [comment]
Restarted the build.

**Comment by starspire** [comment]
@pjedrzejewski I think this pr is ready now :smile:

**Comment by michalmarcinkowski** [comment]
Thanks Krzysztof!

**Pull request merged**